kickstart/scripts/samba-deploy.sh

```diff
@@ -11,6 +11,18 @@ deploy_samba_ubuntu() {
   chown nobody:nogroup /opt/data/fieldpapers
   chmod 777 /opt/data/fieldpapers
 
+  mkdir -p /opt/data/backups/omk
+  chown nobody:nogroup /opt/data/backups/omk
+  chmod 777 /opt/data/backups/omk
```

Comment:
great question for @mojodna; I gave each directory the same rights as https://github.com/AmericanRedCross/posm-build/pull/36/commits/13ec0a8c77276d955e365207ece0578accbc70d6